src/views/HomeView.vue

```diff
@@ -42,18 +72,114 @@
 </template>
 
 <script>
-import { mapGetters } from 'vuex';
+import _ from "lodash";
+import { mapGetters } from "vuex";
+
 export default {
   computed: {
-    ...mapGetters(['appData'])
+    ...mapGetters({
+      appData: "appData",
+      latestPosts: "blog/getLatestPosts"
```

Comment:
this allows you to modify the name of the computed property to be referenced in the vue html. here we change `blog/getLatestPosts` tp `latestPosts`.

Also as a side note, I recently encountered this at work but the general convention for naming vuex store getters is to not use the "get" prefix. The get prefix should only be used when performing a fetch or getting data from somewhere. Here we are simply returning existing data in the store and not performing and manipulation of it.